### ISSUE TITLE
Unable to Create user with name or bytes

### ISSUE THREAD

**Issue by niknoi** [murka/outlinevpn-api#10]
Hi, i cannot create user with name or some params 

For example how i can see the fix
```
public async createUser(params?: Partial<Omit<User, 'id'>>): Promise<User> {
        const response = await this.fetch({
            url: `${this.apiUrl}/access-keys`,
            method: 'POST',
            body: JSON.stringify(...params)
        })

        if(response.ok) {
            const json = JSON.parse(response.body)
            return json
        } else {
            throw new Error('No server found')
        }
    }
```

**Comment by murka** [comment]
Hi, I fix that, update a package to `3.0.0` version, and use: https://github.com/murka/outlinevpn-api?tab=readme-ov-file#create-access-key

so, FYI, `createUser` is deprecated, use `client.createAccessKey ` instead. 


UPD: the `3.0.0` version of package is fully capable of outline-server api

**Comment by niknoi** [comment]
Thanks for updates
